frontend/src/assets/css/variables.scss

```diff
@@ -4,6 +4,7 @@
   --black: #000000;
   --gray-100: #e2e3e5;
   --gray-200: #6c707a;
+  --gray-250: #484b53;
```

Comment:
let's either use gray-300, all we need to switch them to have gray-300, gray-400

Reply:
did I understand you correctly: change current gray-300 to gray-400 and replace in all styles to it, and change gray-250 to gray-300?

Reply:
yep